ush/wdqms.py

```diff
@@ -0,0 +1,766 @@
+import sys
+import os
+import argparse
+import logging
+import pandas as pd
+import numpy as np
+from netCDF4 import Dataset
+from pathlib import Path
+
```

Comment:
Can not load the import command in the parallel run.

Reply:
Add 
```
#!/usr/bin/env python3
```
On the first line.